packages/amplify-category-function/src/provider-utils/awscloudformation/index.ts

```diff
@@ -210,13 +210,13 @@ export async function updateLayerResource(
       projectName: context.amplify.getProjectDetails().projectConfig.projectName,
     };
   }
-  await serviceConfig.walkthroughs.updateWalkthrough(context, undefined, parameters);
+  parameters = await serviceConfig.walkthroughs.updateWalkthrough(context, undefined, parameters);
```

Comment:
I think we dont have to return parameters from this function as we are already passing parameters as a reference in this function

Reply:
I think this is better because it makes it obvious that parameters is being modified